src/state/apis/zapper/zapperApi.ts

```diff
@@ -246,13 +246,20 @@ export const zapperApi = createApi({
           }
         }
 
-        const parsedData = data.map(v2NftItem => {
-          // Actually defined since we're passing supported EVM networks AccountIds
```

Comment:
Wrong assumption since Zapper supports more EVM chains than we do - and unfortunately no, it's not possible to filter out by *multiple* networks at least in a single request, it *is* possible to filter out by a single network but it means we would then need multiple requests to achieve the same here so meh